action/src/index.ts

```diff
@@ -390,7 +390,8 @@ export const main = async ({
       };
     }
     return {
-      commitMessage: commit.commit.message,
+      // Use slice to remove the trailing newline
+      commitMessage: commit.commit.message.slice(0, -1),
```

Comment:
Rather than relying on the last character being a newline (which may change in isomorphic git at some point), let's explicitly trim newline characters only, and all of them at that.

```suggestion
      commitMessage: commit.commit.message.trim(),
```